package/samplers/cmamae/README.md

```diff
@@ -27,6 +27,9 @@ with improvement ranking, all wrapped up in a
 However, it is possible to implement many variations of CMA-MAE and other
 quality diversity algorithms using pyribs.
 
+For visualizing the results of the `CmaMaeSampler`, note that we use the
+`plot_grid_archive_heatmap` function from the `plot_pyribs` plugin.
```

Comment:
@nabenabe0928 What is the correct way to link between plugins in OptunaHub? i.e., How can I provide a link to the `plot_pyribs` plugin here? Similarly, in the `plot_pyribs` plugin, I want to provide a link to the `CmaMaeSampler`.

Reply:
I will quickly merge this PR, so you can surely open another PR later, but basically, the URL for each package is published like [this](https://hub.optuna.org/samplers/cmamae/), so you could link these URLs to your package!